packages/model-viewer/src/model-viewer-base.ts

```diff
@@ -271,7 +288,12 @@ export default class ModelViewerElementBase extends UpdatingElement {
         }
       }, {
         root: null,
-        rootMargin: '10px',
+        // We used to have margin here, but it was causing animated models below
+        // the fold to steal the frame budget. Weirder still, it would also
+        // cause input events to be swallowed, sometimes for seconds on the
+        // model above the fold, but only when the animated model was completely
+        // below. Setting this margin to zero fixed it.
+        rootMargin: '0px',
```

Comment:
You mean it would cause animated models _more than 10px below the fold_ to steal frame budget? If so, this probably warrants some investigation and possibly a Chrome bug...

Reply:
No, it was only when they were within 10px of the fold. Stealing frame budget in that case was expected, but the part where it would swallow input events for long periods is a Chrome bug, I think.